samples/dashboards/demo/personal-portfolio/demo.css

```diff
@@ -0,0 +1,101 @@
+@import url("https://code.highcharts.com/css/highcharts.css");
+@import url("https://code.highcharts.com/dashboards/css/dashboards.css");
+@import url("https://code.highcharts.com/dashboards/css/datagrid.css");
+
+/* Dashboards */
+
+:root {
+    --highcharts-color-0: #c963ff;
+    --highcharts-color-1: #30cbea;
+}
+
+.dashboard-description {
+    background-color: var(--highcharts-neutral-color-5);
+    color: var(--highcharts-neutral-color-100);
+    margin: 0;
+    padding-left: 20px;
+}
+
+.dotted-line {
+    stroke-dasharray: 2, 12;
+}
+
+.highcharts-gauge-chart .highcharts-grid-line {
+    stroke-width: 0;
+}
+
+.highcharts-datagrid-table {
+    border: 0;
+}
+
+#kpi-gauge-risk {
+    flex: 0 1 37.5%;
+}
+
+.highcharts-yaxis .highcharts-tick {
+    stroke-width: 4px;
+    stroke: var(--highcharts-background-color);
+}
+
+.highcharts-radial-axis-grid {
+    stroke-width: 0;
+}
+
+.highcharts-dashboards-component-kpi-value {
+    font-size: 1.8rem;
+    font-weight: 600;
+}
+
+.highcharts-dashboards-component-title {
+    font-size: 0.9rem;
+    font-weight: 500;
+}
+
+.highcharts-datagrid-table tbody td {
+    padding-left: 14px;
+}
+
+@media (max-width: 800px) {
```

Comment:
In addition to this, I think we should add another media queries for the smallest screens (max-width: 500px or even less). The text is overflowing in some of the KPIs, which we can simply resolve with a smaller font size. For the Goal probability, maybe shift the data label up.
<img width="376" alt="Skjermbilde 2024-12-06 kl  07 17 11" src="https://github.com/user-attachments/assets/df54813d-842f-4076-91c9-443f9425430a">
<img width="382" alt="Skjermbilde 2024-12-06 kl  07 17 21" src="https://github.com/user-attachments/assets/1feefa1d-14b3-4cc5-9574-3cc6280f8ceb">